docs/resources/etc_fstab.md.erb

```diff
@@ -0,0 +1,126 @@
+---
+title: About the etc_fstab Resource
+---
+
+# etc_fstab
+
+Use the `etc_fstab` InSpec audit resource to test information about all partitions and storage devices on a system.
+## Syntax
+
+An etc_fstab rule specifies a device name, its mount point, its mount type, the options its mounted with,
+its dump options, and the order the files system should be checked.
+
+## Syntax
+
+Use the where clause to match a property to one or more rules in the fstab file.
+
+    describe etc_fstab.where { device_name == 'value' } do
+      its ( 'mount_point' ) { should eq ['hostname'] }
+      its ( 'file_system_type' ) { should eq [['list']] }
+      its ( 'mount_options' ) { should eq [['list']] }
+      its ( 'dump_options' ) { should eq [['list']] }
+      its ( 'file_system_options' ) { should eq [['list']] }
+    end
+
+Use the optional constructor parameter to give an alternative path to fstab file
+
+    describe etc_fstab(hosts_path).where { device_name == 'value' } do
+      its ( 'mount_point' ) { should eq ['hostname'] }
+      its ( 'file_system_type' ) { should eq [['list']] }
+      its ( 'mount_options' ) { should eq [['list']] }
+      its ( 'dump_options' ) { should eq [['list']] }
+      its ( 'file_system_options' ) { should eq [['list ']] }
+    end
+
+where
+
+* `device_name` is the name associated with the device.
+* `mount_point` is the default directory the device will be mounted if one is not specified in the mount command.
+* `file_system_type` is the type of file system of the device or partition.
+* `mount_options` is the options for the device or partition.
+* `dump_options` is a number used by dump to decide if a file system should be backed up.
+* `file_system_options` is a number that specifies the order the file system should be checked.
+
+## Property Examples and Return Types
+
+### device_name(String)
```

Comment:
I don't know that we document the Ruby type that would be returned for a method in any other docs. Is there a reason you included it here? Also, this is kind of inaccurate since it will be an Array of Strings. I don't think our users need to know that though.